config/connection.js

```diff
@@ -7,9 +7,9 @@ const connection = mysql.createConnection({
     host: "localhost",
     port: 3306,
     user: "root",
-
+    password: "Root1234!",
     //adding process.env (accessing the environment variable password) so I don't have to show my password in code ("PASSWORD= ___ node server.js")
-    password: process.env.PASSWORD,
+    //password: process.env.PASSWORD,
```

Comment:
same thing with the password stuff. When you use `process.env.PASSWORD` here, when you type into terminal   you will enter `PASSWORD=yourpassword node server.js ` instead of just `node server.js`

if you send your code to master, then when everyone else pulls master to their machine, their code won't run because you pushed your password - and we all have different passwords. Hope that makes sense!